Geometry/TrackerCommonData/plugins/dd4hep/DDTIBLayerAlgo.cc

```diff
@@ -406,40 +406,38 @@ static long algorithm(Detector& /* description */,
     Volume dohmCarrier = ns.addVolumeNS(Volume(name, solid, ns.material(dohmCarrierMaterial)));
     int primReplica = 0;
     int auxReplica = 0;
-#if 0
-    for ( size_t i = 0; i < placeDohm*dohmList.size(); i++ )   {
-      double phi   = (std::abs(dohmList[i])+0.5-1.)*dphi;
-      double phix  = phi + 90_deg;
-      double theta = 90_deg;
-      double phiy  = phix + 90._deg;
-      dohmRotation = makeRotation3D(theta, phix, theta, phiy, 0., 0.);
-
-      int    dohmReplica = 0;
+
+    for (size_t i = 0; i < placeDohm * dohmList.size(); i++) {
+      double phi = (std::abs(dohmList[i]) + 0.5 - 1.) * dphi;
+      double phix = phi + 90_deg;
+      double phideg = convertRadToDeg(phix);
+      if (phideg != 0) {
```

Comment:
I think what you want here is a check like:
```
if (std:abs(phix) > tolerance)
```
where `tolerance` is some small constant close to zero.
The line `double phideg = convertRadToDeg(phix);` serves no purpose except to waste CPU, since nothing is done with `phideg` except to check if it's not zero.

Reply:
I’ll leave it to @vargasa

Reply:
@cvuosalo I think you are correct, given the size of the PR this might be optimized at a later stage